cmds/xbmc.py

```diff
@@ -710,7 +719,7 @@ def _files(bot,args,dirs,pid):
   # set last_played for bookmarking
   bot.last_played = (pid,matches[0])
 
-  return msg+'Playlist from "'+match+'" starting with #'+str(num+1)
+  return msg+'Playlist starting with #'+str(num+1)
```

Comment:
Do you mind clarifying a bit more on malformed searches? If the song(s) searched for aren't able to be added to the playlist, then it will return an error earlier in the code. Unless you mean the end user mistypes their search and this was as a call and response mechanism to confirm what was to be played?

I reworded this specific bit as it didn't quite go with playing your entire library.